app/views/labware/_relatives_list.html.erb

```diff
@@ -1,27 +1,31 @@
 <%# locals: (presenter:) %>
+<%
+  found_parents = presenter.labware.parents&.filter{ |parent| parent.purpose.present? }
+  found_children = presenter.labware.children&.filter{ |child| child.purpose.present? }
```

Comment:
Might it be worth naming these `valid_parents` and `valid_children` rather than `found_`?